paasta_tools/utils.py

```diff
@@ -1786,6 +1789,22 @@ def get_auto_hostname_unique_size(self) -> int:
     def get_api_endpoints(self) -> Mapping[str, str]:
         return self.config_dict['api_endpoints']
 
+    def get_api_ca_certificates(self) -> Mapping[str, str]:
+        """ A mapping of cluster name to CA certificate for validating API server identity """
+        return self.config_dict.get('api_ca_certificates', {})
+
+    def get_default_api_ca_certificate(self) -> str:
+        """ A default CA certificate to fall back to if there is not a cluster mapped in
+        get_api_ca_certificates
+        """
+        return self.config_dict.get('default_api_ca_certificate', '/etc/paasta/pki/ca.crt')
+
+    def get_enable_client_cert_auth(self) -> bool:
+        """
+        If enabled present a client certificate from $HOME/.paasta.crt nad $HOME/.paasta.key
```

Comment:
filenames are different from the ones which are used in the code above.

Reply:
Good spot!